### PR TITLE
Add Square workshop event

### PR DESCRIPTION
**Pull request by bdamerac**

### CODE REVIEW

File: content/S2016/S16-event-square.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Food will be available, including vegetarian, vegan, and gluten-free
+options.
+
+There is an attendance cap of 70 people (50 undergrads and 20 grads),
+so make sure to [register]( https://www.eventbrite.com/e/square-tech-workshop-becoming-ok-with-okhttp-tickets-25360009471)
+Note:This event is [women-only](http://wics.uwaterloo.ca/faq)
+
+## Event Details ##
+
++ **Who:** For [Cis & trans women and non-binary individuals](http://wics.uwaterloo.ca/faq)
```
Comment (evykassirer) [inline content/S2016/S16-event-square.md]: You don't need "For"

**Comment by evykassirer** [comment]
Yay I'm glad you can do git from your computer now :D 
That means all the committee are able to contribute the website whee

You can see this change live here: https://wics-site-pr-210.herokuapp.com/

File: content/S2016/S16-event-square.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Square is holding a technical workshop on their open source HTTP & HTTP/2
+client for Android and Java applications.
+Join us to learn more about Okhttp and get a chance to speak with their
+engineers!
+Food will be available, including vegetarian, vegan, and gluten-free
+options.
+
+There is an attendance cap of 70 people (50 undergrads and 20 grads),
+so make sure to [register]( https://www.eventbrite.com/e/square-tech-workshop-becoming-ok-with-okhttp-tickets-25360009471)
```
Comment (evykassirer) [inline content/S2016/S16-event-square.md]: there's a space before "https://www.eventbrite" do you see it?

Comment (evykassirer) [inline content/S2016/S16-event-square.md]: and it does wrap, but it'll wrap even if you put it on a new line and then this line would be so short, so idk if it's worth putting the link in its own line

**Comment by evykassirer** [comment]
(explained that when changes happen in the same branch they go to the same pull request)

**Comment by evykassirer** [comment]
this looks good to me, but another pair of eyes from committee would help me feel more comfy merging ^^

File: content/S2016/S16-event-square.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Square is holding a technical workshop on their open source HTTP & HTTP/2
+client for Android and Java applications.
+Join us to learn more about Okhttp and get a chance to speak with their
+engineers!
+Food will be available, including vegetarian, vegan, and gluten-free
+options.
+
+There is an attendance cap of 70 people (50 undergrads and 20 grads),
+so make sure to [register]
+(https://www.eventbrite.com/e/square-tech-workshop-becoming-ok-with-okhttp-tickets-25360009471)
```
Comment (evykassirer) [inline content/S2016/S16-event-square.md]: oh oops - you can't break up links like that
see https://wics-site-pr-210.herokuapp.com/

**Comment by SrishtiG** [comment]
Looks good to me too! :+1:

File: content/S2016/S16-event-square.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+Square is holding a technical workshop on their open source HTTP & HTTP/2
+client for Android and Java applications.
+Join us to learn more about Okhttp and get a chance to speak with their
+engineers!
+Food will be available, including vegetarian, vegan, and gluten-free
+options.
+
+There is an attendance cap of 70 people (50 undergrads and 20 grads),
+so make sure to [register](https://www.eventbrite.com/e/square-tech-workshop-becoming-ok-with-okhttp-tickets-25360009471)
```
Comment (ehashman) [inline content/S2016/S16-event-square.md]: Missing period at the end of this line, and also the next.

File: content/S2016/S16-event-square.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+Note:This event is [women-only](http://wics.uwaterloo.ca/faq)
```
Comment (ehashman) [inline content/S2016/S16-event-square.md]: Missing a space after Note:

**Comment by fboxwala** [comment]
lgtm

**Pull request merged**